src/torchmetrics/metric.py

```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import builtins
```

Comment:
```suggestion
# It is needed to distinguish between native float and Metric's' function called float.
# later, this function was used instead of the built-in float type...
import builtins
```